pkg/scheduler/core/generic_scheduler.go

```diff
@@ -456,6 +456,8 @@ func (g *genericScheduler) numFeasibleNodesToFind(numAllNodes int32) (numNodes i
 		return minFeasibleNodesToFind
 	}
 
+	klog.V(2).Infof("Get # of evaluated node. Total nodes %v, percentageOfNodesToScore %v, adaptivePercentage %v, node to evaluate %v",
```

Comment:
for small cluster this log will not shown. but for our current investigation, it is OK.